app/models/Classroom.coffee

```diff
@@ -167,10 +169,15 @@ module.exports = class Classroom extends CocoModel
       currentLevel = courseLevels.models[currentIndex]
       currentPlaytime = levelSessionMap[currentLevel.get('original')]?.get('playtime') ? 0
       needsPractice = utils.needsPractice(currentPlaytime, currentLevel.get('practiceThresholdMinutes')) and not currentLevel.get('assessment')
-      nextIndex = utils.findNextLevel(levels, currentIndex, needsPractice)
-    nextLevel = courseLevels.models[nextIndex]
-    nextLevel = arena if levelsLeft is 0
-    nextLevel ?= _.find courseLevels.models, (level) -> not levelSessionMap[level.get('original')]?.get('state')?.complete
+      unless utils.voyagerCourseIDs.includes(courseID)
+        nextIndex = utils.findNextLevel(levels, currentIndex, needsPractice)
+    if utils.voyagerCourseIDs.includes(courseID)
+      # assuming that there will be only one next level in voyager v1 for now
+      nextLevel = new Level(voyagerUtils.findNextLevels(sessions, courseLevels.models)[0])
```

Comment:
In  previous comments I have mentioned the single next level assumption. That is due to it being new code that we want to write realistic tests for. If we lean too heavily into our nextLevel assumption. The tests will end up reflecting that and it won't be easy  to change.

However. This code is now interacting with the existing legacy system. I think it's excellent to leave a comment here about the assumption and only assign the one next level.
:clap: